packages/lab/src/list-next/ListNextContext.tsx

```diff
@@ -0,0 +1,18 @@
+import { createContext } from "@salt-ds/core";
+import { SyntheticEvent, useContext } from "react";
+
+export interface ListItemNextContextValue {
+  disabled: boolean;
+  id?: string;
+  select: (event: SyntheticEvent<HTMLLIElement>) => void;
+  isSelected: (id: string) => boolean;
+  isFocused: (id: string) => boolean;
+}
+
+export const ListItemNextContext = createContext<
```

Comment:
should the file name be ListItemNextContext ?

Reply:
or I feel like rather, this should be ListemNextContext, given it's used in ListNext?